aerogear/digger-jenkins
# Sync jenkinsfiles

# Changes

Adds ansible playbook to fetch digger-jenkins repo and apply Jenkinsfiles changes on app templates.

## Verification steps:

* Run `ansible-playbook localhost.yml`
* Check if templates are using the correct Jenkinsfile.

Note: I am using the `shell` module for file operations since I had a lot of issues with it (pyhton selinux intermittent problem).
